src/schema/README.md

```diff
@@ -369,6 +369,7 @@ The namespaces are:
 | --------------------------- | ----------------------------------------------------------------------------------- | ---------------- |
 | `objects.common_principles` | Terms that are used throughout BIDS                                                 | General terms    |
 | `objects.modalities`        | Broad categories of data represented in BIDS, roughly matching recording instrument | General terms    |
+| `objects.metaentities`      | Placeholders and wildcards to reduce verbosity of some templates in BIDS            | General terms    |
```

Comment:
I agree that these concepts do not fit elsewhere, but I'm not sure that `metaentity` is right. I'm +1 on meta, -1 on entity. Not sure what would be better just now.

Reply:
Happy to take any suggestions :)

Reply:
@effigies - what do you think? Some new name comes up?